design/FY2016/NFP-EcomonizerIntegration.md

```diff
@@ -0,0 +1,1137 @@
+
+#### Field: Minimum Temperature at Cooling Coil Outlet Node
+
+In order to prevent cooling coil freezing, the minimum temperature at the cooling coil outlet is required. The default value is 7.2C.
+
+
+Below is an example input for a Mixed Air Setpoint Manager.
+
+```idf
+SetpointManager:SingleZone:Reheat, ! establishes the setpoint at the system outlet node
+           Supply Air Temp Manager,   !- Name
+           Temperature,   !- Control Variable
+           13.,  !- Minimum Supply Air Temperature
+           45.,  !- Maximum Supply Air Temperature
+           NORTH ZONE,  !- Control Zone Name
+           Zone 3 Node,  !- Zone Node Name
+           Zone 3 Inlet Node,  !- Zone Inlet Node Name
+          Supply Air Temp Nodes;   !- Setpoint Node or NodeList Name
+
+
+SetpointManager:MixedAir, ! uses the system outlet setpoint to establish the mixed air setpoint
+           Mixed Air Temp Manager 1,   !- Name
+           Temperature,   !- Control Variable
+           Air Loop Outlet Node,                   !- Reference Setpoint Node Name
+           Heating Coil Air Outlet Node,   !- Fan Inlet Node Name
+           Air Loop Outlet Node,                   !- Fan Outlet Node Name
+           Mixed Air Nodes;                             !- Setpoint Node or NodeList Name
+
+```
+### UnitarySystemPerformance:Multispeed
+
+#### Field: Name
+
+This alpha field contains the identifying name for the multispeed performance specification.
+
+#### Field: Number of Speeds for Heating
+
+This field defines the number of heating speeds for the heat pump, and must match the number of heating speeds defined in the associated heating coil. The value for this input field defines the number of airflow rate ratios that must be defined for heating in the fields below. The minimum value for this field is one and the maximum value is the number specified in the coil object. If the heating coil type used in the unitary system object is not a multispeed coil type, then this field should be 1.
+
+#### Field: Number of Speeds for Cooling
+
+This field defines the number of cooling speeds for the heat pump, and must match the number of cooling speeds defined in the associated DX cooling coil. The value for this input field defines the number of airflow rate ratios that must be defined for cooling in the fields below. The minimum value for this field is one and the maximum value is the number specified in the coil object. If the cooling coil type used in the unitary system object is not a multispeed coil type, then this field should be 1.
+
+#### Field: Single Mode Operation
+
+This field specifies the coil operation mode for multiple speed DX cooling and heating coils. The allowed choice is Yes or No. The No choice allows a coil works between two adjacent speeds when a system load is greater than the coil capacity at speed 1. The Yes choice allows a coil works with a single capacity at a different speed. The speed number is determined by a system load.  
```

Comment:
@lgu1234 Just to clarify.  This applies to each HVAC time step?  If that's correct, then I would suggest add to the first sentence " . . . and heating coils during each HVAC timestep."